make_wininsts.bat

```diff
@@ -1,10 +1,10 @@
-SET PATH=C:\Python26;%PATH%
```

Comment:
Add ?w=1 to the diff url to remove whitespace-only changes. Compare:
https://github.com/pyepics/pyepics/pull/35/files?w=1
with
https://github.com/pyepics/pyepics/pull/35/files